### DISCUSSION TITLE
CIS-1.8 release date ?

### DISCUSSION THREAD

**Discussion by andypitcher** [aquasecurity/kube-bench#1524] [Questions and Help] [answered]
Hi,

CIS Workbench released CIS-1.8.0 (k8s 1.26) on Oct 6th 2023, Do you have a release date for this version ? 

Thanks.

**Comment by mozillazg** [comment, marked as answer]
I'll add it after https://github.com/aquasecurity/kube-bench/pull/1484 and https://github.com/aquasecurity/kube-bench/pull/1523 are merged. Perhaps we will release it mid-next month.

**Reply by andypitcher** [reply]
Thanks @mozillazg ! If you need some help with https://github.com/aquasecurity/kube-bench/pull/1523 let me know.
